pepper-apis/src/main/java/org/broadinstitute/ddp/thread/ThreadFactory.java

```diff
@@ -0,0 +1,24 @@
+package org.broadinstitute.ddp.thread;
+
+/**
+ * Simple {@link java.util.concurrent.ThreadFactory} that lets
+ * you set the name for the group and the priority.
+ */
+public class ThreadFactory implements java.util.concurrent.ThreadFactory {
```

Comment:
Brought over from Birch--nice to de-propritize slack thread stuff so that pepper can continue doing "real work" when it's crunch time.